packages/xo-server-audit/src/index.js

```diff
@@ -55,6 +55,22 @@ const DEFAULT_BLOCKED_LIST = {
 }
 
 const LAST_ID = 'lastId'
+
+// Db {
+//   lastId: String,
```

Comment:
The types in lowercase.

Reply:
Why?

Reply:
To avoid bike-shedding over this, IMHO, we should simply use TypeScript syntax: https://learnxinyminutes.com/docs/typescript/